packages/xml-views-validation/src/validators/elements/unknown-tag-name.ts

```diff
@@ -237,10 +237,16 @@ function shouldIgnoreElement(xmlElement: XMLElement): boolean {
   // We allow the tag sap.ui.core:FragmentDefinition at the root of the document
   // See: https://sapui5.hana.ondemand.com/1.71.14/#/topic/23b9c779c2274213a281c1fc46b4962b
   // TODO: This should probably only be allowed in fragment.xml files
+
+  // We allow the tag sap.ui.core:ExtensionPoint except on the root of the document
+  // See https://ui5.sap.com/#/topic/403c050da4ae4566b6aafec2bc590389
   return (
-    xmlElement.parent.type === "XMLDocument" &&
-    resolveXMLNS(xmlElement) === CORE_NS &&
-    xmlElement.name === "FragmentDefinition"
+    (xmlElement.parent.type === "XMLDocument" &&
```

Comment:
please don't duplicate the whole condition.
Two of three parts of the condition are identical.